### PR TITLE
OSX: minor readme adjustment

### LINKED ISSUE #1684: 3.6.0: update OSX readme
The [OSX port readme](https://github.com/adventuregamestudio/ags/blob/master/OSX/README.md) contains at least some obsolete information. For instance, it sais that it *"uses a modified Allegro 4 library which sets up an offscreen buffer that the game renders to"*. Since 3.6.0 everything uses SDL2 instead.

It's possible that there are other incorrect data there, as the engine building has changed since.

### PR DESCRIPTION
**Pull request by ericoporto**
adjustments to fix #1684

future things for MacOS would be

- uploading the engine binary to CI artefacts
- some way to [build from editor](https://github.com/adventuregamestudio/ags/issues/1333#issuecomment-1042790885)
- updating the xcode projects

### CODE REVIEW

File: OSX/README.md [hunk 1]
```diff
@@ -27,12 +22,7 @@ TIP: Use `touch AGS.app` to force macos to reload the application icon.
 
 ## Adding resources for the game
 
-All game resources are taken from `OSX/Resources` . Please feel free to replace the stub files that
-exist there with your own game files, as long as they have the same names.
-
-The default accepted name for the game file to be placed there is `game.ags`, which is generated in the `Compiled/Data/`
```
Comment (ivan-mogilko) [inline OSX/README.md:33]: Please, keep a mention of `Compiled/Data/` folder. I think it should say that normally game files are taken from Compiled/Data (where *.ags is a main game data file), and more files could be added if necessary for mac version of the game.

Comment (ericoporto) [reply OSX/README.md:33]: Added the following sentence, what do you think?

> When you build your game with the Editor, your own game `.ags` and additional files should be in `Compiled/Data/`.

Comment (ivan-mogilko) [reply OSX/README.md:33]: My impression from this sentence is that it does not make it clear that the files are taken *from* there, it sounds more like user is supposed to place them there themselves.

Comment (ericoporto) [reply OSX/README.md:33]: switched the order of the sentences to be more obvious, do you think it works better like this?

> After you build your game with the Editor, your own game `.ags` and additional files will be generated be in the `Compiled/Data/` of your game project directory.
> 
> In this port, all game resources are taken from `OSX/Resources`, remove the stub files that exist there and replace with your own game files taken from `Compiled/Data/`.

Comment (ivan-mogilko) [reply OSX/README.md:33]: This may be fine; there's just one typo in the sentence "files will be generated **be** in the"

Comment (ericoporto) [reply OSX/README.md:33]: thanks! Took the opportunity to fix some other grammar errors I haven't thought about to check!

**Pull request merged**